lib/service/Service.ts

```diff
@@ -1,11 +1,27 @@
-import BtcdClient from '../chain/BtcdClient';
+import Logger from '../Logger';
+import BtcdClient, { BtcdInfo } from '../chain/BtcdClient';
+import LndClient, { LndInfo } from '../lightning/LndClient';
+
+type walliInfo = {
+  version: string,
+  btcdInfo: BtcdInfo,
+  lndInfo: LndInfo,
+};
 
 class Service{
-  constructor(private btcdClient: BtcdClient) {}
 
-  public getInfo = async (): Promise<object> => {
-    const info = await this.btcdClient.getInfo();
-    return info;
+  constructor(private logger: Logger, private btcdClient: BtcdClient, private lndClient: LndClient) {}
+
+  public getInfo = async (): Promise<walliInfo> => {
+    const version = require('../../package.json').version;
```

Comment:
This should be moved outside of the function and class so that `package.json` is only read once (when importing the file) instead of every time `getInfo` is called